src/components/ReminderCtaConfirmation.tsx

```diff
@@ -66,17 +95,8 @@ export const ReminderCtaConfirmation = ({
                 </Button>
             </div>
             <StraightLines />
-            {remindMeConfirmationHeaderText && (
-                <h4 css={successHeadingStyles}>{remindMeConfirmationHeaderText}</h4>
-            )}
-            <p css={successTextStyles}>
-                {remindMeConfirmationText} You can manage your email preferences in the My Account
-                area,{' '}
-                <a href="https://manage.theguardian.com/email-prefs">
-                    emails and marketing section
-                </a>
-                .
-            </p>
+            <h4 css={successHeadingStyles}>Something went wrong</h4>
+            <p css={successTextStyles}>Sorry, we were not able to sign you up for a reminder.</p>
```

Comment:
Maybe there's a different name we could use for `successTextStyles` since these definitely aren't success messages!

Reply:
There's a lot of names we can change - particularly for css-related code. This will become particularly important for when we try to thread this functionality into the styleable banner code. Will investigate.

Reply:
File has been removed as we no longer need it